tests/Frameworks/WordPress/Version_4_8/wp-content/themes/twentyseventeen/functions.php

```diff
@@ -565,3 +565,5 @@ function twentyseventeen_front_page_template( $template ) {
  * SVG icons functions and filters.
  */
 require get_parent_theme_file_path( '/inc/icon-functions.php' );
+
+remove_filter('template_redirect', 'redirect_canonical');
```

Comment:
What's the purpose of this one?

Reply:
By default WordPress redirects URL's that don't end with a slash to the slashed version. Our web tests test for endpoints sans trailing slashes `/simple`,  `/simple_view`, and  `/error` which were all returning 301 redirects without this.